### PR TITLE
[ps/es] Added initial version of powershell Spanish guide (based on the latest English version)

### PR DESCRIPTION
**Pull request by alexitosrv**
- [ ] Added powershell-es.html.markdown

### CODE REVIEW

File: es-es/powershell-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,328 @@
+---
+category: tool
+tool: powershell
+contributors:
+    - ["Wouter Van Schandevijl", "https://github.com/laoujin"]
+translators:
+    - ["Alexander Salamanca", "https://github.com/alexitosrv"]
+filename: LearnPowershell-es.ps1
```
Comment (vendethiel) [inline es-es/powershell-es.html.markdown]: Please add "lang: es-es"

File: es-es/powershell-es.html.markdown [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+try {} catch {} finally {}
+try {} catch [System.NullReferenceException] {
+	echo $_.Exception | Format-List -Force
+}
+
+
+### Proveedores
+# Lista de archivos y directorios en la ubicación actual
+ls # o el alias `dir`
+cd ~ # ir a la ruta inicial
```
Comment (vendethiel) [inline es-es/powershell-es.html.markdown]: El directorio principal del usuario

File: es-es/powershell-es.html.markdown [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+# O más powershellezco
+'cadena'.ToUpper() -replace 'E', 'eee'
+
+# Si ¿no recuerda como es que se llama cierto método .NET?
+'cadena' | gm
+
+# Sintaxis para ejecutar métodos .NET estáticos
+[System.Reflection.Assembly]::LoadWithPartialName('Microsoft.VisualBasic')
+
+# Note que las funciones .NET DEBEN ser llamadas usando paréntesis
```
Comment (vendethiel) [inline es-es/powershell-es.html.markdown]: .NET DEBEN?

File: es-es/powershell-es.html.markdown [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+	New-Item -Type file -Path $Profile -Force
+	notepad $Profile
+}
+# Más información en: `help about_profiles`
+# Para un shell más productivo, asegúrese de verifivar el proyecto  PSReadLine descrito abajo
+```
+
+Proyectos interesantes
+
+* [Channel9](https://channel9.msdn.com/Search?term=powershell%20pipeline#ch9Search&lang-en=en) Tutoriales de PowerShell
```
Comment (vendethiel) [inline es-es/powershell-es.html.markdown]: Please mark those with a "(EN)" tag

**Comment by alexitosrv** [comment]
All the comments made by vendethiel were addressed in the last commit.

**Comment by vendethiel** [comment]
:+1: do you still want to make some changes, or is it okay to merge now?

**Comment by alexitosrv** [comment]
For me, it's fine to do the merge as it is.

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**